tests/strategy/failing_strategy.py

```diff
@@ -0,0 +1,87 @@
+
+# --- Do not remove these libs ---
+from freqtrade.strategy.interface import IStrategy
+from pandas import DataFrame
+# --------------------------------
+
+# Add your lib to import here
+import talib.abstract as ta
+
+import nonexiting_module  # noqa
+
+
+# This class is a sample. Feel free to customize it.
+class TestStrategyLegacy(IStrategy):
+    """
+    This is a test strategy using the legacy function headers, which will be
+    removed in a future update.
+    Please do not use this as a template, but refer to user_data/strategy/sample_strategy.py
+    for a uptodate version of this template.
+    """
```

Comment:
Personally i would remove 99% of the things in this class (especially comments  / docstrings) - we can leave the functions with a `pass` or returning the input again ... 

It'll fail at line 10 - nothing after that will matter